### PR TITLE
update title

### PR DESCRIPTION
**Pull request by titojankowski**
As recommended by Dan Walsh - meta guidelines

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -19,7 +19,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>CarbonDoomsDay</title>
+    <title>Carbon Doomsday - Climate change chart & carbon dioxide levels</title>
```
Comment (lwm) [inline public/index.html]: I think `&` needs to be `&amp;` (rusty HTML hat on).

Comment (titojankowski) [inline public/index.html]: indeed! https://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp

**Comment by lwm** [comment]
:+1:

**Comment by lwm** [comment]
First commits @titojankowski! :beers: :rainbow_flag: :beers: :rainbow_flag:

**Comment by titojankowski** [comment]
woot woot!

**Pull request merged**